src/Hosting/Hosting/src/Internal/HostingRequestFinishedLog.cs

```diff
@@ -1,24 +1,28 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
 using System.Collections;
 using System.Collections.Generic;
 using System.Globalization;
-using Microsoft.AspNetCore.Http;
 
 namespace Microsoft.AspNetCore.Hosting.Internal
 {
+    using static HostingRequestStartingLog;
```

Comment:
More @pranavkm points 😄 I see this was introduced "way back" in C# 6, but I haven't seen this used much.